### PR TITLE
Background notifications fixed

### PR DESCRIPTION
**Pull request by LJMNZS**
Closes issue #60 
Background notifications should be working on all browsers
Clicking the notification should bring the website back into focus
The notification should disappear after 5 seconds
Audio notifications not in this PR, still have to decide what sound to use

### CODE REVIEW

File: source/js/scripts/app.js [hunk 1]
```diff
@@ -36,6 +36,10 @@ import * as backend from '../backend.js';
  * }
  */
 
+// Icon assets
+const pomoIcon = require('url:../../img/green-tomato.png');
+const breakIcon = require('url:../../img/red-tomato.png');
```
Comment (markbussard) [inline source/js/scripts/app.js:41]: Can we not use module imports here? I.e., `import pomoIcon from '../../img/green-tomato.png'`?

Comment (alanlwang1) [reply source/js/scripts/app.js:41]: Probably, I'll look into it

**Comment by markbussard** [comment]
There seems to be an issue with the background notification and something being undefined.

Screenshot from 2022-05-23 19-55-05


Also, there may be an issue with the service worker registration?

Screenshot from 2022-05-23 19-55-00

**Comment by markbussard** [comment]
Although when running this branch locally, it is working just fine.

**Comment by alanlwang1** [comment]
Yeah, its not working for me either, I'm not sure what the problem is. 
I'm also getting this error for some reason:
image

**Comment by markbussard** [comment]
> Yeah, its not working for me either, I'm not sure what the problem is. I'm also getting this error for some reason: image
> 
> Is it an issue with firebase? Maybe we can try remaking the PR?

Yeah you can go ahead and try that if you want. I'm not sure either, because everything works just fine locally. The window opens / focuses correctly as well.

**Pull request closed without merging**